nest-forms-backend/src/utils/global-services/mailgun/mailgun.service.ts

```diff
@@ -18,7 +26,12 @@ import {
 export default class MailgunService {
   mailgunClient: IMailgunClient
 
-  constructor() {
+  oloTransporter: nodemailer.Transporter
```

Comment:
a.k.a. garbage truck

Reply:
In next refactor we can rename it to `oloGarbageTruck`, I am in.